metricbeat/module/system/service/service.go

```diff
@@ -50,17 +57,39 @@ func init() {
 // interface methods except for Fetch.
 type MetricSet struct {
 	mb.BaseMetricSet
-	conn     *dbus.Conn
-	cfg      Config
-	unitList unitFetcher
+	conn            *dbus.Conn
+	cfg             Config
+	unitList        unitFetcher
+	instanceTracker map[string]insntanceTracker
+}
+
+func defaultConfig() Config {
+	return Config{
+		StateFilter:         []string{},
+		PatternFilter:       []string{},
+		TrackNonService:     false,
+		TrackNotFound:       false,
+		TrackInstantiated:   false,
+		InstantiatedTimeout: time.Second * 60,
+	}
+}
+
+// in certain versions of systemd, dead-inactive instantiated services will disappear
```

Comment:
Nit. Godocs use to start with the name of the documented object.
```suggestion
// instanceTracker keeps track of instances of systemd units.
// In certain versions of systemd, dead-inactive instantiated services will disappear
```